OloEditor/assets/models/Sponza/README.md

```
@@ -0,0 +1,92 @@
+I also repacked the separate metallic/roughness textures into the glTF layout (G - roughness, B - metallic).
+The images are also re-encoded as PNG instead of TGA.
+All the materials also had a constant diffuse factor of about 0.58. I assume it was supposed to be there, so I kept it.
+I also ran the vertices and indices through a mesh optimizer.
+
+## Licensing notes
+
+On 2023-03-08, the model was confirmed to be [available](https://www.cryengine.com/marketplace/product/crytek/sponza-sample-scene)
+
+Taken from copyright.txt in SponzaPBR\_Textures.rar
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)